docusaurus.config.js

```diff
@@ -110,18 +110,109 @@ const config = {
         hideOnScroll: true,
         items: [
           {
-            type: "docSidebar",
-            sidebarId: "docSidebar",
+            type: "dropdown",
+            label: "Product",
             position: "left",
-            label: "Docs",
-            to: "/docs/getting-started",
+            items: [
+              {
+                label: "White Paper",
+                to: "/whitepaper",
+              },
+              {
+                label: "vs MinIO",
+                to: "/blog/comparisons/computer-vision/iot/performance-comparison-reductstore-vs-minio",
+              },
+              {
+                label: "vs MinIO & InfluxDB",
+                to: "/blog/comparison/iot/reductstore-benchmark",
+              },
+              {
+                label: "vs TimescaleDB",
+                to: "/blog/comparisons/iot/reductstore-vs-timescaledb",
+              },
+              {
+                label: "vs MongoDB",
+                to: "/blog/comparisons/iot/reductstore-vs-mongodb",
+              },
+              {
+                label: "Schedule a Technical Review",
```

Comment:
I'm not sure it is about product.

Reply:
@atimin checkout TimescaleDB, they added this button on their product tab. Could be good to get more people to send us an email. Or maybe a button somewhere else?

Reply:
Agree, don't have a better idea so far. Let's keep it here for now.